src/d_main.c

```diff
@@ -182,6 +182,13 @@ int eventhead, eventtail;
 //
 void D_PostEvent(event_t *ev)
 {
+  if (ev->type == ev_mouse && localview.active)
+  {
+    M_InputTrackEvent(ev);
+    G_Responder(ev);
```

Comment:
As I understand it, `G_Responder` should not be entered between ticks at all. The idea is to disable interpolation and only move the viewpoint in the renderer without changing the imput. Or is that not possible?

Reply:
I'm concerned with that too. A reduced `G_ResponderMouse()` that only updates the viewpoint variables seems safer. I'll take a closer look later.